### PR TITLE
update doc to v3.1.0

### PR DESCRIPTION
**Pull request by kqzh**

### CODE REVIEW

File: README-CN.md [hunk 1]
```diff
@@ -10,9 +10,11 @@
 
 一些收集的 metrics 是：
```
Comment (MegaByte875) [inline README-CN.md:11]: 语句不通

File: README-CN.md [hunk 2]
```diff
@@ -69,12 +71,12 @@ scrape_configs:
 ```shell
 $ docker run -d --restart=always --name nebula-stats-exporter -p 9100:9100 \
     -v "<<PATH_OF_CONFIG_FILE>>:/config.yaml" \
-    vesoft/nebula-stats-exporter:v0.0.6 --bare-metal --bare-metal-config=/config.yaml
+    vesoft/nebula-stats-exporter:v3.1.0 --bare-metal --bare-metal-config=/config.yaml
```
Comment (MegaByte875) [inline README-CN.md:74]: use kernel release version?

File: README-CN.md (hunk 1 above)
Comment (kqzh) [reply README-CN.md:11]: 改成 `目前收集的 metrics 是` ?

File: README-CN.md (hunk 2 above)
Comment (kqzh) [reply README-CN.md:74]: yes, because release-tag will use v3.1.0, I think unite is better

File: README-CN.md (hunk 1 above)
Comment (MegaByte875) [reply README-CN.md:11]: ok

File: README-CN.md (hunk 2 above)
Comment (veezhang) [reply README-CN.md:74]: Is the version definition strictly consistent, or just the first two bits? 

For examples, 

1. nebula release v3.1.0
2. stats release v3.1.0
3. you find bug in stats, then stats release v3.1.1
4. nebula release v3.1.1

So what version is the stats?

**Review by veezhang — APPROVED** [review]
LGTM

File: README-CN.md (hunk 2 above)
Comment (kqzh) [reply README-CN.md:74]: if stats don't have change, stats release v3.1.1, otherwise v3.1.2, promise x.y compatible nebula

**Pull request merged**